### DISCUSSION TITLE
使用 NDI Tools 向虚拟机中的腾讯会议共享宿主机屏幕

### DISCUSSION THREAD

**Discussion by giscus** [kxxt/kxxt-website#153] [Thread]
# 使用 NDI Tools 向虚拟机中的腾讯会议共享宿主机屏幕

鉴于本人日常将腾讯系软件安装在虚拟机中，然而在虚拟机中使用腾讯会议时无法共享宿主机的屏幕，经过几番周折之后最终使用 NDI Tools 解决了这个问题，于是写下此文，方便有同样需求的人参考。

https://www.kxxt.dev/blog/using-ndi-to-share-screen-with-vm/

**Comment by eric-gitta-moore** [comment]
感谢
